src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -162,10 +164,9 @@ function MoneyRequestView({report, parentReport, shouldShowHorizontalRule, polic
             </OfflineWithFeedback>
             <OfflineWithFeedback pendingAction={lodashGet(transaction, 'pendingFields.merchant') || lodashGet(transaction, 'pendingAction')}>
                 <MenuItemWithTopDescription
-                    description={translate('common.merchant')}
+                    description={isDistanceRequest ? translate('tabSelector.distance') : translate('common.merchant')}
```

Comment:
Actually that would break [this code](https://github.com/Expensify/App/blob/arosiclair-distance-request-details/src/components/TabSelector/TabSelector.js#L67) so we should probably not move it. We could add another entry under `common` but duplicating it doesn't feel right.

Reply:
@arosiclair Can you please create a follow-up issue to clean this up?

Reply:
Done: https://github.com/Expensify/App/issues/25722